Update influxdb-rails to master branch

ChrisBr opened the pull request:
To not store cache_hits and counts for render_collection series anymore.
These values have a high cardinality and therefore not usable as tags
which causes high load on metrics.o.o due to constantly indexing.

https://github.com/influxdata/influxdb-rails/pull/63

coolo commented:
Or apply the patch in bundled-gems.spec - but the gem service doesn't like git it seems.

ChrisBr commented:
> Nice fix in the gem tada , but the packages seems to not have permissions to download gems:

Hm, that's weird, it should work as this was one of the reasons why we implemented the ``cpio`` mode based on bundler in the gem service :thinking: I will talk with @M0ses about that as we should fix it ...

> I am also not sure if we should download it. What about opening a new release wanted/needed issue in influxdb-rails?

I already talked with the maintainer but he wants to add more features before releasing a new version ...

ChrisBr commented:
> Hm, that's weird, it should work as this was one of the reasons why we implemented the cpio mode based on bundler in the gem service

Ok, I changed now to use an URL instead of git, this is anyway better because ``github:`` does not support SSL (https://bundler.io/guides/git.html#security). Furthermore I locked the commit now.

ChrisBr commented:
I debugged with @M0ses and it seems that the service container only has network to the geminabox container but not to the internet. We will change this next year but for now it is not supported ... Let's wait for the release next week, closing...
